src/pybind/mgr/feedback/module.py

```diff
@@ -0,0 +1,110 @@
+
+"""
+Feedback module
+
+See doc/mgr/feedback.rst for more info.
+"""
+
+from mgr_module import CLIReadCommand, HandleCommandResult, MgrModule
+import errno
+from .feedbackServices import CephTrackerClient
+from .feedbackModel import Feedback
+
+
+class FeedbackModule(MgrModule):
+
+    # there are CLI commands we implement
+    @CLIReadCommand('set issue_key')
+    def set_api_key(self, key: str) -> HandleCommandResult:
+        """
+        Set Ceph Issue Tracker API key
+        """
+        try:
+            self.set_store('issue_tracker', key)
+        except Exception as error:
+            return HandleCommandResult(stderr=f'Exception in setting API key : {error}')
+        return HandleCommandResult(stdout="Successfully updated key")
+
+    @CLIReadCommand('delete issue_key')
+    def delete_api_key(self) -> HandleCommandResult:
+        """
+        Delete Ceph Issue Tracker API key
+        """
+        try:
+            self.set_store('issue_tracker', None)
+        except Exception as error:
+            return HandleCommandResult(stderr=f'Exception in deleting API key : {error}')
+        return HandleCommandResult(stdout="Successfully deleted key")
+
+    def is_api_key_set(self):
+        try:
+            key = self.get_store('issue_tracker')
+            if key is None:
+                return False
+            return key != ''
+        except:
+            return False
+
+    @CLIReadCommand('get issue_key')
+    def get_api_key(self) -> HandleCommandResult:
+        """
+        Get Ceph Issue Tracker API key
+        """
+        try:
+            key = self.get_store('issue_tracker')
+            if key is None:
+                return HandleCommandResult(stderr='Issue tracker key is not set. Set key with `ceph set issue_key <your_key>`')
+        except Exception as error:
+            return HandleCommandResult(stderr=f'Error in retreiving issue tracker API key: {error}')
+        return HandleCommandResult(stdout=f'Your key: {key}')
+
+    @CLIReadCommand('get issue')
+    def get_issue(self, issue_number: int) -> HandleCommandResult:
+        """
+        Fetch an issue
+        """
+        try:
+            issue_number = int(issue_number)
+        except TypeError:
+            return -errno.EINVAL, '', f'Invalid issue number {issue_number}'
+        cephTrackerClient = CephTrackerClient()
```

Comment:
In Python (unlike JS/Typescript) CameCase is only used for ClassNames, not instances, which are in snake_case:
```suggestion
        ceph_tracker_client = CephTrackerClient()
```

Reply:
@cypherean I see that some of the sugesstions are not yet addressed in backend as well, can u please address them and if they are already done can you mark those conversations as resolved